ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -109,7 +109,7 @@ def _write_timing_tree(self) -> None:
         timing_path = f"{self.summaries_dir}/{self.run_id}_timers.json"
         try:
             with open(timing_path, "w") as f:
-                json.dump(get_timer_tree(), f, indent=2)
+                json.dump(get_timer_tree(), f, indent=4)
```

Comment:
Changed to match the output of `json.tool` (which I was using to format C# output)